### PR TITLE
feat(style guide): updated UI pathing

### PR DESCRIPTION
**Pull request by akristen**
I used a second table because it was easier for a first draft. Open to other ways of organizing the information.

Looking for other examples of UI pathing conventions, too.

### CODE REVIEW

**Review by austin-schaefer — DISMISSED** [review]

File: src/content/docs/style-guide/formatting/ui-paths.mdx [hunk 1]
```diff
@@ -139,14 +110,82 @@ Our goal for UI paths is to make them easy to use and understand, preferably wri
...
+        3. Select a span to see its details.      
+    </td>
+    </tr>
+
+    <tr>
+      <td>
+        For lengthier paths, simplify by bracing the UI elements with `>`. Bold everything in the path.
+      </td>
+      <td>
+          Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) APM & services > (select an app) > Transactions > (select a transaction) > (select a transaction trace) > Trace details**
```
Comment (akristen) [inline src/content/docs/style-guide/formatting/ui-paths.mdx:167]:
```suggestion
          Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & services > (select an app) > Transactions > (select a transaction) > (select a transaction trace) > Trace details**
```

File: src/content/docs/style-guide/formatting/ui-paths.mdx [hunk 2]
```diff
@@ -139,14 +110,82 @@ Our goal for UI paths is to make them easy to use and understand, preferably wri
...
+    </tr>
+
+        <tr>
+      <td>
+        For multi-step procedures, use an ordered list with full sentences.
+      </td>
+      <td>
+        To see details for a specific span:
+
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM**.
```
Comment (akristen) [inline src/content/docs/style-guide/formatting/ui-paths.mdx:154]:
```suggestion
        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & services**.
```

File: src/content/docs/apm/agents/manage-apm-agents/agent-data/agent-attributes.mdx [hunk 3]
```diff
@@ -150,7 +150,7 @@ Request attributes are associated with specific transaction traces, browser trac
 />
 
 <figcaption>
-  **APM > (select an app) > Monitoring > Transactions > (select a transaction) > (select a trace):** Here is an example from the Ruby agent of the **Request attributes** and **Custom attributes** for a transaction trace. Exact attributes depend on your agent and on your attribute configuration.
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **APM & services > (select an app) > Monitoring > Transactions > (select a transaction) > (select a trace):** Here is an example from the Ruby agent of the **Request attributes** and **Custom attributes** for a transaction trace. Exact attributes depend on your agent and on your attribute configuration.
```
Comment (austin-schaefer) [inline src/content/docs/apm/agents/manage-apm-agents/agent-data/agent-attributes.mdx:153]: Double-bolded here:

```** > **APM & services ```

File: src/content/docs/style-guide/formatting/ui-paths.mdx [hunk 4]
```diff
@@ -139,14 +110,82 @@ Our goal for UI paths is to make them easy to use and understand, preferably wri
...
+        Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app)**.      
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+## Use your best judgment [#judgment]
 
-For example, whether or not to include a URL is up to you. If including `Go to one.newrelic.com` in a path description is cumbersome or unnatural, exclude it. If it helps orient the reader, feel free to include it. This same thinking applies towards most of our guidelines.
+If you’re ever feeling stuck when writing a UI path, use your best judgment. The best way to format or word a UI path may depend on the path’s length and context.
```
Comment (a-sassman) [inline src/content/docs/style-guide/formatting/ui-paths.mdx:191]:
```suggestion
If you’re ever feeling stuck when writing a UI path, use your best judgment. The best way to format or word a UI path may depend on the procedure's length and context.
```

**Review by a-sassman — APPROVED** [review]
LGTM! Just left some minor suggestions - take em or leave em!

**Pull request merged**